code/modules/power/apc.dm

```diff
@@ -430,11 +425,11 @@
 	if(stat & (NOPOWER | BROKEN))
 		return
 	if(!second_pass) //The first time, we just cut overlays
-		addtimer(CALLBACK(src, .get_spooked, TRUE), 1)
+		addtimer(CALLBACK(src, /obj/machinery/power/apc/proc.get_spooked, TRUE), 1)
```

Comment:
Is this the correct way to do it? proc.get_spooked I mean.

Reply:
Unfortunately it is. Dont worry, I hate it just as much as you.